### ISSUE TITLE
Bug on SQL Server 

### ISSUE THREAD

**Issue by uyab** [efriandika/laravel-settings#5]
``` bash
[Illuminate\Database\QueryException]   SQLSTATE[HY000]: General error: 2714 General SQL Server error: Check messages from the SQL Server [2714] (severity 16) [(null)] (SQL: create table "settings_local" ("key" nvarchar(100) not null, "value" nvarchar(max) null))
```

**Comment by efriandika** [comment]
it has been solved in [v1.2.0](https://github.com/efriandika/laravel-settings/releases/tag/v1.2.0)

**Comment by efriandika** [comment]
check 92cbd77c337a9512be0893b04ee59b2a41225ade for more info
